lib/db/schemaupdater.go

```diff
@@ -835,6 +836,43 @@ func (db *schemaUpdater) dropIndexIDsMigration(_ int) error {
 	return db.dropIndexIDs()
 }
 
+func (db *schemaUpdater) fixRecvEncFileSize(_ int) error {
+	meta := newMetadataTracker(db.keyer, db.evLogger)
+	t, err := db.newReadWriteTransaction()
+	if err != nil {
+		return err
+	}
+	defer t.close()
+
+	var key []byte
+	for _, folderStr := range db.ListFolders() {
+		folder := []byte(folderStr)
+		var innerErr error
+		err := t.withHave(folder, protocol.LocalDeviceID[:], nil, false, func(fi protocol.FileIntf) bool {
+			f := fi.(protocol.FileInfo)
+			meta.removeFile(protocol.LocalDeviceID, f)
+			f.Size -= int64(f.ProtoSize())
+			meta.addFile(protocol.LocalDeviceID, f)
```

Comment:
I don't get it, how is this specific to the affected files, or even specific to encrypted fileInfos?

Reply:
It's not. I need to stop rushing PRs... Will fix.

Reply:
If it's easy to do so... otherwise I'm inclined to let this be a "Drop the folder and resync if it matters to you" issue, given the files on disk are also wrong(-ish)